src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/ExpectedOutputs/OneRunWithInvocationAndNotifications.sarif

```diff
@@ -27,7 +27,6 @@
           "message": "Exception evaluating rule \"C2152\". Rule disabled; run continues.",
           "level": "error",
           "threadId": 52,
```

Comment:
>threadId": 52, [](start = 11, length = 14)

case 4 of 4 - missing test cases which were added in this pr. Does the missing "time" value make sense? did we remove such a property from 2.x?